custom-images/docker-compose.override.yml

```
@@ -28,16 +28,21 @@ services:
     volumes:
       - ${LOCAL_DATA_PATH}\mssql:c:\data
   
+  # Mount our Solr data folder and use our retagged Solr image.
   solr:
-    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-solr:${VERSION:-latest}
+    volumes:
+      - type: bind
+        source: .\docker\data\solr
+        target: c:\data
```

Use our LOCAL_DATA_PATH here?
```suggestion
      - ${LOCAL_DATA_PATH}\solr:c:\data
```

---

done!